<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgRegister</class>
 <widget class="QDialog" name="CDlgRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDlgRegister</string>
  </property>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRegister">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmail">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CDlgRegister</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
